pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -134,27 +158,13 @@ public void WriteXml(XmlWriter writer)
 
         private enum EL
         {
-            document_hash
+            document_hash,
+            en_root_hash
```

Comment:
Let's drop the en_ part. It is not relevant that English is included in the hash. We have no ja_root_hash or any other language.